.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,101 @@
+name: PHP Coding Standards
+
+on:
+    push:
+        branches:
+            - trunk
+            - 'release/**'
+            - 'wp/**'
+    pull_request:
+        paths:
```

Comment:
I'm not sure whether limiting this by `paths` is a good idea. There is no `composer.lock` file (good, in this case), but the version constraints in the `composer.json` file are not that tightly bound, so a new release of one of the CS dependencies will now not trigger a build, while it probably should.

Reply:
Fixed in 1d10c2c294cad20dcea822e785bc1d88c357420c